.kokoro/tests/run_tests.sh

```diff
@@ -67,8 +72,8 @@ for file in **/requirements.txt; do
     cd "$file"
 
     # If $DIFF_ONLY is true, skip projects without changes.
-    if [[ "$ONLY_DIFF" = "true" ]]; then
-        git diff --quiet origin/master.. .
+    if [[ "DIFF_FROM" != "" ]]; then
```

Comment:
`${DIFF_FROM}` instead of `DIFF_FROM`

* need `$`
* nit: also better to use braces

Reply:
Good catch, thank you